docker-compose.yml

```diff
@@ -71,7 +92,6 @@ services:
     image: postgres
     ports:
       - "5432:5432"
-    restart: always
```

Comment:
seems to have no effect?

Reply:
Is this a question or did you research this?

Reply:
I researched this and asking myself why it is there :P

Reply:
I researched this some more - it does not have the effect that I expected, it only restarts terminated containers (but not unhealthy ones). Documented that in the comment for the autoheal container

Reply:
Weird, I would have expected it to restart unhealthy containers. Good finding.